ksqldb-functional-tests/src/test/java/io/confluent/ksql/test/driver/KsqlTesterTest.java

```diff
@@ -210,9 +211,11 @@ private void execute(final ParsedStatement parsedStatement) {
       return;
     }
 
+    // TODO: what's a cleaner way around this failure?
+    final ConfiguredStatement<?> injected = new DefaultFormatInjector().inject(configured);
```

Comment:
Same problem as with the QTT framework above: this is needed or else tests that don't explicitly specify `KEY_FORMAT` in CS/CT statements fail. Feels hacky to add the injector here but I can't think of a better solution (besides requiring explicit `KEY_FORMAT` in all tests, which doesn't seem right).

Reply:
@agavra is probably the best person to comment here...